bin/compile

```diff
@@ -261,7 +277,7 @@ ensureGo() {
         if [ -d "${cache}/go-path" ]; then
```

Comment:
You are going to need to move these lines into `resetCache` as they guard against the read only nature of the go modules cache